src/Authoring/WinRT.SourceGenerator/DiagnosticUtils.cs

```diff
@@ -121,6 +100,34 @@ public void FindDiagnostics()
             } 
         }
 
+        private void IgnoreCustomTypeMappings(INamedTypeSymbol typeSymbol, 
+            ref IEnumerable<MethodDeclarationSyntax> methods, 
+            ref IEnumerable<PropertyDeclarationSyntax> properties)
+        {
+            string QualifiedName(INamedTypeSymbol sym) 
+            { 
+                return sym.OriginalDefinition.ContainingNamespace + "." + sym.OriginalDefinition.MetadataName; 
+            }
+
+            bool SameProp(string a, string b)
+            {
+                if (a.StartsWith("get_") || a.StartsWith("put_"))
+                {
+                    return a.Substring(4) == b;
+                }
+                return false;
+            }
+
+            foreach (var @interface in typeSymbol.AllInterfaces.
+                        Where(symbol => MappedCSharpTypes.Contains(QualifiedName(symbol)) ||
+                                        ImplementedInterfacesWithoutMapping.Contains(QualifiedName(symbol))))
+            {
+                var mems = @interface.GetMembers();
+                methods = methods.Where(m => !mems.Where(mem => mem.Name == m.Identifier.Text).Any());
```

Comment:
I feel like you would want a symbol comparison here as what happens if 2 functions have same name, but have a diff # of parameters and one is from a custom type mapping while the other isn't.

Reply:
addressed